src/main/java/io/vertx/mqtt/MqttEndpoint.java

```diff
@@ -232,6 +272,16 @@
   @Fluent
   MqttEndpoint publishReceivedHandler(Handler<Integer> handler);
 
+  /**
+   * Set the pubrec handler on the MQTT endpoint. This handler is called when a PUBREC
+   * message is received by the remote MQTT client
+   *
+   * @param handler the handler
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  MqttEndpoint publishReceivedMessageHandler(Handler<MqttPubRecMessage> handler);
```

Comment:
I'm wondering if we should not bame it `PubRecMesssageHandler` for consistency

Reply:
What should we do with `MqttEndpoint publishReceivedHandler(Handler<Integer> handler);` then?

Reply:
right this is a good point.